lib/shelley/test/data/cardano-node-shelley/alonzo-genesis.yaml

```diff
@@ -1,7 +1,7 @@
 # Taken from https://hydra.iohk.io/build/6782523/download/1/index.html
 # and not sanity-checked for appropriateness for when we turn on Alonzo
 # in our test cluster.
-adaPerUTxOWord: 0
+adaPerUTxOWord: 34482
```

Comment:
Does the comment above still apply?

Reply:
Ah, forgot about it! Thanks, will remove.

Reply:
214bc4b404d6707b99fb8fca461a6eb7843bd3f2 Plutus script parameters probably need to be sanity checked at some point, so kept a comment about that.